uefi-std-example/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "uefi-std-example"
+description = """
+
+"""
```

Comment:
nit: can drop the empty description field, not required for an unpublished package.